httpexpect/importAPI/get_job_test.go

```diff
@@ -1,68 +1,117 @@
 package importAPI
 
 import (
-	"fmt"
 	"net/http"
+	"os"
 	"testing"
 
+	"github.com/ONSdigital/dp-api-tests/testDataSetup/mongo"
+	"github.com/ONSdigital/go-ns/log"
 	"github.com/gavv/httpexpect"
 	. "github.com/smartystreets/goconvey/convey"
+	mgo "gopkg.in/mgo.v2"
 )
 
-// Get a job
 // Get information about a single job
-// 200 - Return a single jobs information
-func TestGetJob_ReturnsSingleJob(t *testing.T) {
+// 200 - Return a single job information
```

Comment:
I don't think these comments are needed before test functions; the name of the function explains exactly what it will be testing.